internal/cli/atlas/security/ldap/status_test.go

```diff
@@ -23,14 +23,14 @@ import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
 	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
 	"github.com/mongodb/mongodb-atlas-cli/internal/test"
-	"go.mongodb.org/atlas/mongodbatlas"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 func TestVerifyStatus_Run(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockStore := mocks.NewMockLDAPConfigurationDescriber(ctrl)
 
-	var expected *mongodbatlas.LDAPConfiguration
+	var expected *atlasv2.NDSLDAPVerifyConnectivityJobRequest
```

Comment:
I will use this PR as example on why we need to have separate model names instead of classes.